### PR TITLE
[Update] chore/upgrade-to-source

### PR DESCRIPTION
**Pull request by dselgo**
qdt-components recently had a major release where they changed their entire code base to use React hooks. I merged their most recent release and modified the files that we have previously modified. The main file that I changed was the components/QdtViz/QdtViz.js file.

### CODE REVIEW

**Comment by dselgo** [comment]
DO NOT MERGE THIS INTO MASTER AFTER REVIEWING! If we merge this into master, then the changes go live. We will hold off on merging until the next Prism release.

**Review by neaslerEidex — APPROVED** [review]

**Pull request merged**
